From 42627c6c772b32d018fee8009f50afa514b6504b Mon Sep 17 00:00:00 2001
From: Kevin Hao <kexin.hao@windriver.com>
Date: Fri, 29 Mar 2013 09:32:19 +0800
Subject: [PATCH] powerpc/t4240: add the reg property for pci bridge device node

The reg property in the pci bridge device node is used to bind this
device node to the pci bridge device. Then all the pci devices under
this bridge could use the interrupt maps defined in this device node
to do the irq translation. So if this property is missed, the pci
traditional irq mechanism will not work.

Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/powerpc/boot/dts/fsl/t4240si-post.dtsi |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/boot/dts/fsl/t4240si-post.dtsi b/arch/powerpc/boot/dts/fsl/t4240si-post.dtsi
index 3719040..9e15cd8 100644
--- a/arch/powerpc/boot/dts/fsl/t4240si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/t4240si-post.dtsi
@@ -65,6 +65,7 @@
 		#size-cells = <2>;
 		#address-cells = <3>;
 		device_type = "pci";
+		reg = <0 0 0 0 0>;
 		interrupts = <16 2 1 15>;
 		interrupt-map-mask = <0xf800 0 0 7>;
 		interrupt-map = <
@@ -91,6 +92,7 @@
 		#size-cells = <2>;
 		#address-cells = <3>;
 		device_type = "pci";
+		reg = <0 0 0 0 0>;
 		interrupts = <16 2 1 14>;
 		interrupt-map-mask = <0xf800 0 0 7>;
 		interrupt-map = <
@@ -117,6 +119,7 @@
 		#size-cells = <2>;
 		#address-cells = <3>;
 		device_type = "pci";
+		reg = <0 0 0 0 0>;
 		interrupts = <16 2 1 13>;
 		interrupt-map-mask = <0xf800 0 0 7>;
 		interrupt-map = <
@@ -143,6 +146,7 @@
 		#size-cells = <2>;
 		#address-cells = <3>;
 		device_type = "pci";
+		reg = <0 0 0 0 0>;
 		interrupts = <16 2 1 12>;
 		interrupt-map-mask = <0xf800 0 0 7>;
 		interrupt-map = <
-- 
1.7.0

